src/js/hooks/cycleCount/useAllProductsTab.jsx

```diff
@@ -94,27 +105,40 @@ const useAllProductsTab = () => {
         </TableCell>
       ),
     }),
-    columnHelper.accessor('product.tags', {
+    columnHelper.accessor((row) =>
+      // eslint-disable-next-line react/no-array-index-key
+      row?.product?.tags?.map((tag, index) => <Badge label={tag} variant="badge--purple" key={index} />), {
```

Comment:
this will have to be fixed after Alan's PR is merged. `tag` is an object, so we would need to pass `tag.name` here and the `key` should never be an index. The key could be `tag.id`.